packages/dds/tree/src/feature-libraries/schemaSummarizer.ts

```diff
@@ -170,6 +170,15 @@ export class SchemaEditor<TRepository extends StoredSchemaRepository>
 		return false;
 	}
 
+	public tryResubmitOp(content: any): boolean {
```

Comment:
`any` is very non-type safe in a way that easily spreads by accident. Avoid using `any` whenever practical.  Prefer `unknown` (or more specific types) for data which an unknown type. Here and in reSubmitCore you can and should probably just use JsonCompatibleReadOnly.

Additionally any code which uses insufficiently specific types should be extra careful to document how it must be used since the types arn't enough.

Reply:
updated to use `JsonCompatibleReadOnly`

Reply:
I mentioned this should also be changed in reSubmitCore.

Reply:
Also, applyStashedOp would be good to fix

Reply:
updated `reSubmitCore` and `applyStashedOp` with `JsonCompatibleReadOnly`